airbyte-integrations/connectors/source-posthog/source_posthog/spec.json

```diff
@@ -0,0 +1,21 @@
+{
+  "documentationUrl": "https://docs.airbyte.io/integrations/sources/posthog",
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Posthog Spec",
+    "type": "object",
+    "required": ["api_key", "start_date"],
+    "additionalProperties": false,
+    "properties": {
+      "start_date": {
+        "type": "string",
```

Comment:
can we add examples, regex and format attributes, see other connectors for example